### ISSUE TITLE
License value 'LicenseRef-SSAL-LICENSE' is invalid

### ISSUE THREAD

**Issue by collimarco** [rubygems/rubygems#5506]
Hello,

We have a gem with a custom license:

```
s.license = 'LicenseRef-SSAL-LICENSE'
```

Then we have a file `SSAL-LICENSE` in the repository.

This is the approach recommended here for custom licenses:
https://guides.rubygems.org/specification-reference/

However we get:

```
WARNING:  license value 'LicenseRef-SSAL-LICENSE' is invalid.  Use a license identifier from
http://spdx.org/licenses or 'Nonstandard' for a nonstandard license.
```

That contradicts your documentation. Can you please remove / fix this warning?

**Comment by simi** [comment]
Hello. Please can you update your post to follow issue template? Have you tried latest RubyGems (as suggested in top of the issue template)?

This seems to be fixed in https://github.com/rubygems/rubygems/commit/6a9b62811d0acecbd4b5ec5d1649fc184e95af17 and being released in RubyGems 3.3.0. Since you have not provided `gem env` as requested in issue template, it is hard to find out what's the cause. If you're running older version, you should be able to upgrade and get rid of this message. If you're running newer version, this is indeed possible bug and we need to fix. But with current info it is impossible to reproduce.

**Comment by collimarco** [comment]
@simi Yes, sorry, I had a previous version. In the most recent version of `gem` I don't get the warning. Great, thanks!
